build.gradle

```diff
@@ -18,8 +18,8 @@ apply plugin: 'org.owasp.dependencycheck'
 
 project.ext {
 
-    CC_VERSION = '8.2.0'
-    MXBUILD_VERSION = '7.23.6.53654'
+    CC_VERSION = '8.3.0'
+    MXBUILD_VERSION = '8.12.0.2541'
 }
 

```

Comment:
Can we update the OWASP dependency checker to org.owasp:dependency-check-gradle:6.0.3 as well?

Reply:
Done.